config/config.go

```diff
@@ -268,7 +283,7 @@ func TestBaseConfig() BaseConfig {
 	cfg.chainID = "tendermint_test"
 	cfg.Mode = ModeValidator
 	cfg.ProxyApp = "kvstore"
-	cfg.FastSyncMode = false
+	cfg.BlockSyncMode = false
```

Comment:
I actually think that this parameter should be removed from the base config and placed in the `BlockSyncConfig` under the field name "enable". This would align nicely with how the `StateSyncConfig` is constructed. If you think now is a good time to make the change then I would advocate for it else we can wait till a later release

Reply:
I'll do a quick pass to see if this is extremely painful to move around. If not, I'll implement, otherwise I'll recommend keeping as is.

Reply:
Sounds good